packages/platform/types/organizations/bookings/get-org-bookings-statistics.input.ts

```diff
@@ -0,0 +1,28 @@
+import { ApiProperty } from "@nestjs/swagger";
+import { Transform } from "class-transformer";
+import { ArrayMinSize, IsArray, IsNumber, IsOptional } from "class-validator";
+
+import { GetBookingsInput_2024_08_13 } from "@calcom/platform-types";
+
+export class GetOrganizationsBookingsStatisticsInput extends GetBookingsInput_2024_08_13 {
```

Comment:
We already have an identical type `GetOrganizationsBookingsInput` in `packages/platform/types/organizations/bookings/get-org-bookings.input.ts` that we can use so just reuse that.